mona_sdk/auth/auth_utils.py

```diff
@@ -0,0 +1,135 @@
+# ----------------------------------------------------------------------------
+#    Copyright 2021 MonaLabs.io
```

Comment:
what's the reason we put this here? I suppose it should either appear once or in all modules, why just these 2?